clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SecurityGroupController.groovy

```diff
@@ -158,9 +158,10 @@ class SecurityGroupController {
                     @PathVariable String region,
                     @PathVariable String securityGroupName,
                     @RequestParam(value = "vpcId", required = false) String vpcId) {
-    def securityGroup = securityGroupProviders.find { secGrpProv ->
-      secGrpProv.cloudProvider == cloudProvider
-    }.get(account, region, securityGroupName, vpcId)
+    def securityGroup = securityGroupProviders.collect { secGrpProv ->
+      secGrpProv.cloudProvider == cloudProvider ? secGrpProv.get(account, region, securityGroupName, vpcId) : null
+    }.findAll({ it })
```

Comment:
Isn't your `collect` + `findAll` a  `findResults`?

Reply:
Ahhh I was doing `findAll` thinking it was supposed to be `findResults`, this is better